app/assets/main/react_components/dashboard/climate_actions/manage_actions/kanban/KanbanActionContainer.jsx

```diff
@@ -190,14 +190,12 @@ const KanbanActionContainer = ({
     });
   };
 
-  const orderBadgesOnItemDragged = (performedCategories, movedItem) => {
-    return orderBy(performedCategories, ({ id }) =>
+  const orderAchievementsOnItemDragged = (achievement, movedItem) => {
```

Comment:
The previous name "performedCategories" implies that there are several, is it one or several achievements?

Reply:
"achievement" or "achievements"?